bundles/org.openhab.binding.boschshc/src/main/resources/Smart Home Controller Issuing CA.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'm assuming you downloaded this from your Smart Home Controller. Do you know that they are the same on all of them or do you expect that we should download them from the SHC in a later version?

Reply:
These two PEM files must be used for all Smart Home Controllers.  Normally there will be only very few changes/updates for the two *.pem files.
I expect a change in about 4-5 years, because the issuing CA PEM expires on 16.08.2025. In case of security issues the PEMs could be changed too before this date by Bosch.